.circleci/config.yml

```diff
@@ -405,11 +405,17 @@ workflows:
           name: "STAGING build container"
           requires:
             - static_tests
-          context: [ deployment-ecs-staging-ynr ]
+          # One might expect to see the staging context here, but the
+          # development one is required because that's the account where the
+          # AWS ECR registry is found
+          context: [ deployment-ecs-development-ynr ]
```

Comment:
What's in the context that's needed to access the registry? I thought it was basically open? Can we move whatever the value is into a SSM param, or do you need the AWS creds here?

Reply:
We use the AWS creds to authenticate with the public registry. Though it's open for pull operations it still requires authentication for push operations (which happen if the build and test are successful)

Reply:
Go it. I expect we're going to want dedicated creds that only have access to push to the registry. Especially if/when we start pushing to a production or shared DC registry. 

I don't think this is needed now though, so I'll make an issue for it for us to do later.